sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -61,8 +61,6 @@
  */
 public final class UnsafeRow extends InternalRow implements Externalizable, KryoSerializable {
 
-  public static final int WORD_SIZE = 8;
-
```

Comment:
This is added by [map_entries](https://github.com/apache/spark/commit/a6e883feb3b78232ad5cf636f7f7d5e825183041#diff-fa494816e13a1d53c1c55af01bc7a396R65)